ghdocs/TUTORIAL.md

```diff
@@ -48,7 +48,7 @@ Now open up your favorite text editor or IDE and create a new file called index.
 
 ## Step 2 – Adding Fabric to a project
 
-Download Fabric, unzip it, and copy **fabric.css** and **fabric.components.css** into your **css/** folder. Then add the following lines to the `<head>` section of your index.html document:
+Go to our release page and [Download the latest version of Fabric](https://github.com/OfficeDev/Office-UI-Fabric/releases/latest), the link is under Downloads. After downloading the zip, extract the contents to a folder. Then go into /dist/css. Copy **fabric.css** and **fabric.components.css** into your  **todo/css/** project folder. Then add the following lines to the `<head>` section of your index.html document:
```

Comment:
I would rephrase this to:
"Go [here to download the latest .zip of Fabric](https://github.com/OfficeDev/Office-UI-Fabric/releases/latest), and then extract the contents. Then go into /dist/css and copy **fabric.css** and **fabric.components.css** into your **todo/css/** project folder. Then add the following lines to the `<head>` section of your index.html document:"